tests/roles/stop_openstack_services/tasks/main.yaml

```diff
@@ -45,7 +45,11 @@
                     "tripleo_ceilometer_agent_compute.service"
                     "tripleo_ceilometer_agent_ipmi.service"
                     "tripleo_ceilometer_agent_notification.service"
-                    "tripleo_ovn_cluster_northd.service")
+                    "tripleo_ovn_cluster_northd.service"
+                    "tripleo_ironic_neutron_agent.service"
+                    "tripleo_ironic_api.service"
+                    "tripleo_ironic_inspector.service"
+                    "tripleo_ironic_conductor.service")
```

Comment:
Can you also add them into the docs?

https://github.com/openstack-k8s-operators/data-plane-adoption/blob/6ff06824e4f4ac62119d6c98e53e3c1418dabab1/docs_user/modules/openstack-stop_openstack_services.adoc#L96

Reply:
Done.